.github/workflows/build/action.yml

```diff
@@ -0,0 +1,36 @@
+---
+
+name: common build tasks
```

Comment:
I wonder whether https://github.com/elastic/apm-agent-java/blob/974ca32266fefb599d5560e93f4dbdce349473fd/.github/workflows/snapshoty.yml could also use this composite action, what do you think?